addons/account/models/chart_template.py

```diff
@@ -100,14 +103,26 @@ class AccountChartTemplate(models.Model):
     account_ids = fields.One2many('account.account.template', 'chart_template_id', string='Associated Account Templates')
     tax_template_ids = fields.One2many('account.tax.template', 'chart_template_id', string='Tax Template List',
         help='List of all the taxes that have to be installed by the wizard')
-    bank_account_code_prefix = fields.Char(string='Prefix of the bank accounts', required=True)
-    cash_account_code_prefix = fields.Char(string='Prefix of the main cash accounts', required=True)
-    transfer_account_code_prefix = fields.Char(string='Prefix of the main transfer accounts', required=True)
+    bank_account_code_prefix = fields.Char(string='Prefix of the bank accounts',
+                                           compute='_compute_bank_account_code_prefix',
+                                           inverse='_inverse_compute_bank_account_code_prefix', store=True,
+                                           required=False, recursive=True)
```

Comment:
You need to fields. One stored and one not stored. Otherwise the same value will be stored on all the records, which we don't want.
You read from the non stored computed field, but you write on the stored non computed field.